generators/generator-base.js

```diff
@@ -2554,6 +2564,11 @@ templates: ${JSON.stringify(existingTemplates, null, 2)}`;
     dest.serviceDiscoveryType = config.serviceDiscoveryType;
 
     dest.embeddableLaunchScript = config.embeddableLaunchScript;
+
+    this.loadDerivedServerConfig(dest, config);
+  }
+
+  loadDerivedServerConfig(dest, config) {
```

Comment:
Probably it's better to change the parameter order to match loadServerConfig api.

Reply:
Ooops